Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'webpacker'
 gem 'devise'
+gem 'active_model_serializers'
```

Comment:
The `ActiveModelSerializers` docs encourage you to specify a version, since there are breaking changes between each one (e.g. `gem 'active_model_serializers', '~> 0.10.0'`.)